### DISCUSSION TITLE
Code Block Content Tabs - Without Indent

### DISCUSSION THREAD

**Discussion by flywire** [facelessuser/pymdown-extensions#1495] [Q&A] [answered]
https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#grouping-code-blocks

> Code blocks are one of the primary targets to be grouped, and can be considered a special case of content tabs...

The blocks are already marked. Why not optionally support existing end of block as end of tab content instead of requiring indenting?

Code blocks are defined by lines before and after the code so standard code can be inserted without changing each line. The issue with current [tabbed](https://facelessuser.github.io/pymdown-extensions/extensions/tabbed/) implementation is each line within the tab must be changed.

**Comment by facelessuser** [comment, marked as answer]
I believe I already answered this, but it is due to the fact that it works better with the Python Markdown parser. I'm also not sure what the complaint is exactly.

Tabs can hold complex content, not just code blocks. Managing this complex content in a container in Python Markdown honestly works best using indentation like lists, Admonitions, and Details are handled. If Python Markdown made it easier to do this without indentation syntax, I may have considered other ways.

I'm not going to go into specifics with why it is easier to handle a Table with a block processor but not a Tabbed container, but as an example, there is a reason why the `fenced_code` extension in Python Markdown doesn't work nested in lists and such and SuperFences does. Also, there is a reason that SuperFences code is so complex and why I don't use that approach with other Markdown constructs.
